### ISSUE TITLE
Accuracy Issue

### ISSUE THREAD

**Issue by PhenomenalOnee** [chenjun2hao/CenterFace.pytorch#40]
Really Nice Tutorial..
I have trained using code from https://github.com/siriusdemon/Build-Your-Own-Face-Model/tree/master/detection centerface detection and facing Accuracy Issue.
Some Modifications I have done according to your repo as follows
HeatMap generation using your code.
FocalLoss implementation for heatmap loss
removed landmark generation layer.
Data augmentation ( random crop, random scale, translation) from https://github.com/Paperspace/DataAugmentationForObjectDetection

I have trained Mobilnet_10 model with pretrained weights upto 90 epochs with LR 5e-4 and facing accuracy problem as compared to original centerface model.
Also during inference if i use threshold to 0.50, 1000+ face detections are made (false detection)
Can you help?

**Comment by Watebear** [comment]
> Really Nice Tutorial..
> I have trained using code from https://github.com/siriusdemon/Build-Your-Own-Face-Model/tree/master/detection centerface detection and facing Accuracy Issue.
> Some Modifications I have done according to your repo as follows
> HeatMap generation using your code.
> FocalLoss implementation for heatmap loss
> removed landmark generation layer.
> Data augmentation ( random crop, random scale, translation) from https://github.com/Paperspace/DataAugmentationForObjectDetection
> 
> I have trained Mobilnet_10 model with pretrained weights upto 90 epochs with LR 5e-4 and facing accuracy problem as compared to original centerface model.
> Also during inference if i use threshold to 0.50, 1000+ face detections are made (false detection)
> Can you help?

Hello, i am reimplementing centerface recently. But the test results is bad. Shall we work together? My email is 540295877@qq.com.

**Comment by PhenomenalOnee** [comment]
@Watebear  yes sure we can . I have mailed you. we can contact and start this journey
